docs/designers-developers/developers/tutorials/block-tutorial/block-controls-toolbar-and-sidebar.md

```diff
@@ -90,13 +90,13 @@ You can also customize the toolbar to include controls specific to your block ty
 ```
 {% ESNext %}
 ```js
-const { registerBlockType } = wp.blocks;
+import { registerBlockType } from '@wordpress.blocks';
```

Comment:
```suggestion
import { registerBlockType } from '@wordpress/blocks';
```

Reply:
Thanks, I knew I was bound to make a mistake with so many changes. :+1: